mysql-test/suite/rocksdb/t/drop_table-master.opt

```diff
@@ -1,4 +1,3 @@
 --rocksdb_max_subcompactions=1
---rocksdb_info_log_level=info_level
```

Comment:
@alxyang Thanks for the confirmation, that's really all I needed to know in order to prioritize it properly on our side.